src/tests/ReactA11yInputElementsRuleTests.ts

```diff
@@ -20,10 +20,21 @@ describe('reactA11yInputElementsRule', (): void => {
         TestHelper.assertViolations(ruleName, script, []);
     });
 
+    it('should pass on input elements without placeholder of type radio, checkbox, file', (): void => {
```

Comment:
> should pass

Should fail, right? Looks like your change works (🎉) but the test is failing.

Reply:
Oh sorry you're right, I had it mixed up. The test is right but the code looks like it has a bug.